solidity/dashboard/src/services/tbtc-authorization.service.js

```diff
@@ -189,17 +190,25 @@ const fetchBondingData = async (web3Context) => {
 
     const operatorBondingDataMap = new Map()
     for (let i = 0; i < createdBonds.length; i++) {
+      const operatorAddress = web3Utils.toChecksumAddress(
+        createdBonds[i].operator
+      )
       const bondedEth = await fetchLockedBondAmount(
         web3Context,
-        createdBonds[i].operator,
+        operatorAddress,
         createdBonds[i].holder,
         createdBonds[i].referenceID
       )
 
-      operatorBondingDataMap.set(
-        web3Utils.toChecksumAddress(createdBonds[i].operator),
-        bondedEth
-      )
+      if (operatorBondingDataMap.has(operatorAddress)) {
```

Comment:
yea, looks good. I think we can simplified just a bit by checking for a _falsy_ bond value, but up to you:
```
      const currentBond = operatorBondingDataMap.get(operatorAddress)
      if (currentBond) {
        operatorBondingDataMap.set(operatorAddress, add(currentBond, bondedEth))
      } else {
        operatorBondingDataMap.set(operatorAddress, bondedEth)
      }
```